jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/pom.xml

```diff
@@ -547,6 +547,7 @@
               <compileSourcesArtifact>org.uberfire:uberfire-preferences-api</compileSourcesArtifact>
               <compileSourcesArtifact>org.uberfire:uberfire-preferences-client</compileSourcesArtifact>
               <compileSourcesArtifact>org.uberfire:uberfire-ssh-api</compileSourcesArtifact>
+              <compileSourcesArtifact>org.uberfire:appformer-js-bridge</compileSourcesArtifact>
```

Comment:
@tiagobento no need to add the dependency to _appformer-js-bridge_ in this showcase?

Reply:
Just added it. Thanks!